crypto/secp256k1/curve_test.go

```
@@ -0,0 +1,1285 @@
+// Copyright (c) 2015-2024 The Decred developers
+// Copyright 2013-2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package secp256k1
+
+import (
+	"fmt"
+	"math/big"
+	"math/bits"
+	"math/rand"
```

golangci-lint (2.12.2)

[error] ST1019: package "math/rand" is being imported more than once

(staticcheck)